docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -186,6 +89,24 @@ Architectural Enclaves is through **Application Enclave Service Manager
 can facilitate various SGX services such as launch approval, remote attestation
 quote signing, etc.
 
+Oasis node supports the (legacy) EPID and (newer) DCAP attestation methods.
+Following instructions differ depending on the attestation method used.
+
+To see if your system supports DCAP attestation run the following:
+
+```bash
+ cpuid -1  | grep "SGX"
+ ```
+
+and look for the following line:
+```
+      SGX_LC: SGX launch config supported      = true
+```
+
+If your system doesn't support the "SGX_LC: SGX launch config supported", skip to the [EPID attestation](#legacy-epid-attestation) section.
+
+## DCAP Attestation
+
 ### Ubuntu 22.04/20.04/18.04
```

Comment:
Should we still show these old Ubuntu versions given that they are not really supported (old kernel, old libc, etc.)? I would remove them.